package.json

```diff
@@ -15,17 +15,19 @@
     "test": "mocha 'spec/javascripts/**/**spec.js?(x)' 'app/javascript/packages/**/*.spec.js?(x)'",
     "es5-safe": "is-es5-safe 'public/packs?(-test)/js/*.js'",
     "normalize-yaml": "normalize-yaml",
-    "build": "true"
+    "prebuild": "find app/assets/builds -type f -not -name '.keep' -delete",
```

Comment:
what if we called this a "clean" step, and had prebuild call clean? A little more indirection but it gives it a clearer name?

Reply:
> what if we called this a "clean" step, and had prebuild call clean? A little more indirection but it gives it a clearer name?

Yep, [I wrote one of those](https://github.com/aduth/is-dependency/blob/4fca0c7222ca98c98020b837045c934b2719d62f/package.json#L7-L9) just the other day, so surprised I didn't opt for it, but I'm all for it 👍

Reply:
Added "clean" script in c314432.